### PR TITLE
add document for server mode

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
Signed-off-by: xiang <xiang13225080@163.com>

### CODE REVIEW

**Comment by WangXiangUSTC** [comment]
@Andrewmatilde @cwen0 PTAL

File: README.md [hunk 1]
```diff
@@ -29,7 +29,13 @@ chmod +x chaosd && mv chaosd /usr/local/bin/chaosd
 
 ## Usage
 
-### Process attack
+You can use Chaosd to inject failures with [Command mode](#command-mode) or [Server mode](#server-mode).
```
Comment (cwen0) [inline README.md:32]: Can we add a catalog in front?

Comment (WangXiangUSTC) [reply README.md]: done

**Comment by cwen0** [comment]
@dcalvin PTAL

**Comment by Andrewmatilde** [comment]
LGTM

File: README.md [hunk 2]
```diff
@@ -29,7 +29,27 @@ chmod +x chaosd && mv chaosd /usr/local/bin/chaosd
...
+    - [Disk attack](#disk-attack-1)
+    - [Recover attack](#recover-attack-1)
+
+### Command mode
+
+Using Chaosd as a command-line tool.
+
+#### Process attack
 
 * kill process
```
Comment (dcalvin) [inline README.md:54]: Could you please add a short description for each attack category and the corresponding sub-tasks?

File: README.md [hunk 3]
```diff
@@ -29,7 +29,27 @@ chmod +x chaosd && mv chaosd /usr/local/bin/chaosd
 
 ## Usage
 
-### Process attack
+You can use Chaosd to inject failures with `Command mode` or `Server mode`:
```
[the remark this answers is not in the document]
Comment (WangXiangUSTC) [reply README.md]: done

File: README.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # chaosd
 
-Chaosd is an easy-to-use Chaos Engineering tool. This tool is used to inject failures to the physic node, such as kill process, network failure, CPU burn, memory burn and etc.
+Chaosd is an easy-to-use Chaos Engineering tool. This tool is used to inject failures to the physic node, such as kill process, network failure, CPU burn, memory burn, etc.
```
[the remark this answers is not in the document]
Comment (WangXiangUSTC) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (WangXiangUSTC) [reply README.md]: done

**Comment by WangXiangUSTC** [comment]
@dcalvin PTAL again

File: README.md [hunk 5]
```diff
@@ -1,10 +1,26 @@
 # chaosd
 
-Chaosd is an easy-to-use Chaos Engineering tool. This tool is used to inject failures to the physic node, such as kill process, network failure, CPU burn, memory burn and etc.
+chaosd is an easy-to-use Chaos Engineering tool used to inject failures to a physical node. chaosd supports the following two modes to inject failures:
```
Comment (dcalvin) [inline README.md:3]:
```suggestion
chaosd is an easy-to-use Chaos Engineering tool used to inject failures to a physical node. Currently, two modes are supported:
```

File: README.md [hunk 6]
```diff
@@ -13,80 +29,322 @@ Before deploying Chaosd, make sure the following items have been installed:
...
 
-* stop process
+Attacks a process according to the PID or process name. Supported tasks are:
 
-```bash
-$ chaosd attack process stop -p [pid] # set pid or pod name
-```
+- **kill process** 
+  
+    Description: Kills a process by sending `SIGKILL` signal
```
Comment (dcalvin) [inline README.md:58]:
```suggestion
    Description: Kills a process by sending the `SIGKILL` signal
```

File: README.md [hunk 7]
```diff
@@ -13,80 +29,322 @@ Before deploying Chaosd, make sure the following items have been installed:
...
+    Attack network successfully, uid: 2c865e6f-299f-4adf-ab37-94dc4fb8fea6
+    ```
 
-* delay network packet
+- **stop process**
 
-```bash
-$ chaosd attack network delay -d eth0 -i 172.16.4.4 -l 10ms
-```
+    Description: Kills a process by sending `SIGKILL` signal
```
Comment (dcalvin) [inline README.md:70]:
```suggestion
    Description: Kills a process by sending the `SIGKILL` signal
```

File: README.md [hunk 8]
```diff
@@ -13,80 +29,322 @@ Before deploying Chaosd, make sure the following items have been installed:
...
+#### Network attack
 
-```bash
-$ chaosd attack network corrupt -d eth0 -i 172.16.4.4 --percent 50%
-```
+Attacks the network using `iptables`, `ipset`, and `tc`. Supported tasks are:
+
+- **delay network packet**
+
+    Description: Sends messages with the specified latency:
```
Comment (dcalvin) [inline README.md:84]:
```suggestion
    Description: Sends messages with the specified latency
```

File: README.md [hunk 9]
```diff
@@ -13,80 +29,322 @@ Before deploying Chaosd, make sure the following items have been installed:
...
+
+    Sample usage:
+
+    ```bash
+    $ chaosd attack network delay -d eth0 -i 172.16.4.4 -l 10ms
+    ```
+
+- **loss network packet**
```
Comment (dcalvin) [inline README.md:92]:
```suggestion
- **lose network packet**
```
Use "lose" to be consistent?

File: README.md [hunk 10]
```diff
@@ -13,80 +29,322 @@ Before deploying Chaosd, make sure the following items have been installed:
...
+>
+> Make sure you are operating with the privileges to run iptables, ipset, etc. Or you can run chaosd with `sudo`.
+
+#### Process attack
+
+Attacks a process according to the PID or process name. Supported tasks are:
+
+- **kill process**
+
+    Description: Kills a process by sending `SIGKILL` signal
```
Comment (dcalvin) [inline README.md:223]:
```suggestion
    Description: Kills a process by sending the `SIGKILL` signal
```

File: README.md [hunk 11]
```diff
@@ -13,80 +29,322 @@ Before deploying Chaosd, make sure the following items have been installed:
...
+    Sample usage:
+
+    ```bash
+    curl -X POST "127.0.0.1:31767/api/attack/process" -H "Content-Type: application/json"  -d '{"process": "{pid}", "signal": 9}' # set pid or pod name
+    {"status":200,"message":"attack successfully","uid":"e6d01a30-4528-4c70-b4fb-4dc47c4d39be"}
+    ```
+
+- **stop process**
+
+    Description: Kills a process by sending `SIGKILL` signal
```
Comment (dcalvin) [inline README.md:234]:
```suggestion
    Description: Kills a process by sending the `SIGKILL` signal
```

File: README.md [hunk 12]
```diff
@@ -13,80 +29,322 @@ Before deploying Chaosd, make sure the following items have been installed:
...
+
+    Description: Sends messages with the specified latency
+
+    Sample usage:
+
+    ```bash
+    $ curl -X POST "127.0.0.1:31767/api/attack/network" -H "Content-Type: application/json"  -d '{"device": "eth0", "ipaddress": "172.16.4.4", "action": "delay", "latency": "10ms", "jitter": "10ms", "correlation": "0"}'
+    ```
+
+- **loss network packet**
```
Comment (dcalvin) [inline README.md:257]:
```suggestion
- **lose network packet**
```

**Review by Andrewmatilde — APPROVED** [review]
LGTM

**Pull request merged**